app/views/miq_request/_prov_field.html.haml

```diff
@@ -48,6 +48,16 @@
     .form-group
       %label.col-md-2.control-label{:valign => "top"}
         = field_hash[:description]
+        - if field_hash[:help]
+          %a{:tabindex => 0,
+             :role             => "button",
+             :"data-toggle"    => "popover",
+             :"data-trigger"   => "focus",
+             :"data-html"      => "true",
+             :title            => "",
+             :"data-content"   => "#{field_hash[:help]}",
+             :"data-placement" => "top"}
+            %span.fa.fa-info-circle
```

Comment:
Seriously? I've been squashing all my commits this whole time!

So for example the latest commit would not be squashed with the previous (even though it's a modification of the previous?)